staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -863,7 +864,9 @@ func (o *ApplyOptions) saveLastApplyAnnotationIfNecessary(
 		info.Name,
 		types.ApplyPatchType,
 		modified,
-		nil,
+		&metav1.PatchOptions{
```

Comment:
This is already passing fieldmanager via `WithFieldManager`

Reply:
In this case the hardcoded value set at https://github.com/kubernetes/kubernetes/pull/126699/files#diff-0d73c392875ae1876a4e74dccf25726c83ad93a850aa2fbd9560438efc467f9aR830

...which I guess is intentional here, since the last-applied annotation is always "managed" by this method.